src/unstructured_client/_hooks/custom/split_pdf_hook.py

```diff
@@ -33,8 +33,13 @@
 PARTITION_FORM_FILES_KEY = "files"
 PARTITION_FORM_SPLIT_PDF_PAGE_KEY = "split_pdf_page"
 PARTITION_FORM_STARTING_PAGE_NUMBER_KEY = "starting_page_number"
+PARTITION_FORM_CONCURRENCY_LEVEL_KEY = "split_pdf_concurrency_level"
 
 DEFAULT_STARTING_PAGE_NUMBER = 1
+DEFAULT_CONCURRENCY_LEVEL = 5
```

Comment:
IMO, I'd like to stick with just "threads" or "concurrency_level". This part is an example of it being slightly confusing to use both terms. 
I personally think "threads" is better/more simple. I also think `split_pdf_concurrency_level` could just be called `threads` or `call_threads`. 
One thing I'm not clear on, though, is if this argument _only_ applies to pdf documents or if it'd also apply to images or other docs.

Reply:
Yeah, I'd also like to choose one name and stick with it. To me "concurrency level" should be clear enough for users. Using "thread" would be a lie since splits are handled by separate processes.

Reply:
@badGarnet @awalker4 @hubert-rutkowski85 @mpolomdeepsense  what's your opinion? 🤔

Reply:
I would lean toward concurrency level:
- this is technically not a threadpool so this is not controlling number of threads, but number of concurrent processes
- when there are issues debugging threads vs. processes are often different
- as a developer (this variable is exposing to developers) being clear on what the variable controls can help more than reducing different names
- with that said in this client we seem to just have processes so we can stick with concurrency level

Reply:
i guess we can resolve this thread